charts/datadog/values.yaml

```diff
@@ -533,6 +533,10 @@ datadog:
         # datadog.apm.instrumentation.language_detection.enabled -- Run language detection to automatically detect languages of user workloads (beta).
         enabled: true
 
+      injector:
+        # datadog.apm.instrumentation.injector.imageTag -- The image tag to use for the APM Injector (preview).
+        imageTag: ""
```

Comment:
does it requires ma minimum agent/cluster-agent version? if yes you can add it in the option description. 

as example https://github.com/DataDog/helm-charts/blob/0b7b146bc4e7728ab881ebc07b6fd340a632db2b/charts/datadog/values.yaml#L138

Reply:
Add based on the changelog for when `DD_APM_INSTRUMENTATION_INJECTOR_IMAGE_TAG` was added: https://github.com/DataDog/datadog-agent/blob/main/CHANGELOG-DCA.rst#7570